backends/arm/test/ops/test_linear.py

```diff
@@ -120,7 +120,7 @@ def _test_linear_tosa_MI_pipeline(
             ArmTester(
                 module,
                 example_inputs=test_data,
-                compile_spec=common.get_tosa_compile_spec(),
+                compile_spec=common.get_tosa_compile_spec(permute_memory_to_nhwc=False),
```

Comment:
I guess rank==4 ==> NHWC breaks this?

Reply:
Yes, (1, ,1, 1, C) works though. As a step towards to solution (2) we plan on removing this flag in the near future.